malaria24/ona/models.py

```diff
@@ -769,6 +769,15 @@ class InboundSMS(models.Model):
     reply_to = models.ForeignKey('SMS', null=True)
 
 
+class SMSEvent(models.Model):
+    """
+    An event for a message sent from the system.
+    """
+    event_type = models.CharField(max_length=255)
```

Comment:
We could maybe have this have the `choices` attribute, since there's a limited list of options that Junebug will return for this field, but not super important.

Reply:
I'd rather leave it as is for now. That way this is agnostic of any additions or deletions (however unlikely) we might make to that list in Junebug later.